installer/conf/container.conf

```diff
@@ -76,7 +76,7 @@
   heartbeat_type tcp
 
   <server>
-    host healthmodel-replicaset-service.kube-system
+    host "#{ENV['REPLICASET_SERVICE_ENDPOINT']}"
```

Comment:
Is this expected to be passed in the yaml?

Reply:
https://github.com/microsoft/OMS-docker/pull/303 is the PR for the yaml change(s) along with the clusterrole name change revert